public_profiler/v3.5-DD/ogl/skills.js

```diff
@@ -473,11 +473,14 @@ var skillsHelpURL = Object.assign(
   extraSpellHelp ? {
     // Feats:
     'expeditious dodge':                    baseExtraHelpURL + 'Expeditious_Dodge',
+    'deadly defense':                       baseExtraHelpURL + 'Deadly_Defense',
 
     // Dervish special abilities.
+    'dervish ac bonus':                     baseExtraHelpURL + 'Dervish#AC_Bonus_.28Ex.29',
     'dervish dance':                        baseExtraHelpURL + 'Dervish#Dervish_Dance_.28Ex.29',
     'movement mastery':                     baseExtraHelpURL + 'Dervish#Movement_Mastery_.28Ex.29',
     'slashing blades':                      baseExtraHelpURL + 'Dervish#Slashing_Blades',
+    'fast movement':                        baseExtraHelpURL + 'Dervish#Fast_Movement_.28Ex.29',
```

Comment:
Open question: should this require a dervish prefix, same as AC Bonus, for likewise being somewhat generic.

Reply:
Fast movement without qualifier should link to https://www.d20srd.org/srd/classes/barbarian.htm#barbarianFastMovement a base class ability that works somewhat differently (notably, stacks with boots).

Reply:
Done.